integration-tests/pom.xml

```diff
@@ -190,7 +190,6 @@
                 <module>hibernate-reactive-postgresql</module>
                 <module>hibernate-reactive-panache</module>
                 <module>hibernate-reactive-panache-kotlin</module>
-                <module>hibernate-reactive-panache-blocking</module>
```

Comment:
What happened to this one?

Reply:
This module was testing the functionality that got removed, see https://github.com/quarkusio/quarkus/blob/2.15.0.Final/integration-tests/hibernate-reactive-panache-blocking/src/main/java/io/quarkus/it/panache/reactive/TestEndpoint.java#L11-L16.